architecture/core_concepts/routes.adoc

```diff
@@ -37,6 +43,29 @@ Configuration] guide for information on deploying a router.
 ====
 endif::[]
 
+A router uses the service selector to find the service and the endpoints backing
+the service. Service-provided load balancing is bypassed and replaced with the
+router's own load balancing. Routers watch the cluster API and automatically
+update their own configuration according to any relevant changes in the API
+objects. Routers may be containerized or virtual. Custom routers can be deployed
```

Comment:
It was in the original. I think you can have a router in a container or not. How would you phrase this? For now I dropped "or virtual"